certutils/certutils.go

```diff
@@ -9,6 +9,27 @@ import (
 	"net/http"
 )
 
+// TLSRPCServerConfig is the configuration you use to create a TLSRPCServer
+type TLSRPCServerConfig struct {
```

Comment:
I don't think we need a new config type.  We can just let the router be nil when used with the RPC constructor.